pyproject.toml

```diff
@@ -3,30 +3,47 @@ requires = ["hatchling"]
 build-backend = "hatchling.build"
 
 [project]
-name = "py-sonar-scanner_DAVID_K"
+# PYSCAN-35: Decide on definitive package name
+name = "py-sonar-scanner"
 dynamic = ["version"]
 authors = [
   { name="Guillaume Dequenne", email="guillaume.dequenne@sonarsource.com" },
+  { name="Jeremi Do Dinh", email="jeremi.dodinh@sonarsource.com" },
   { name="Maksim Grebeniuk", email="maksim.grebeniuk@sonarsource.com" },
   { name="David Kunzmann", email="david.kunzmann@sonarsource.com" },
 ]
 description = "Sonar Scanner for the Python Ecosystem"
 readme = "README.md"
 license = {file = "LICENSE"}
 requires-python = ">=3.8"
+keywords = [
+  "sonar",
+  "sonarqube",
+  "sonarcloud",
+  "cleancode"
+]
 classifiers = [
-    "Programming Language :: Python :: 3",
-    "Operating System :: OS Independent",
+  "Environment :: Console",
+  "Intended Audience :: Developers",
+  "Operating System :: OS Independent",
+  "Programming Language :: Python",
+  "Programming Language :: Python :: 3",
+  "Programming Language :: Python :: 3 :: Only",
+  "Programming Language :: Python :: 3.8",
+  "Programming Language :: Python :: 3.9",
+  "Programming Language :: Python :: 3.10",
+  "Programming Language :: Python :: 3.11",
+  "Programming Language :: Python :: 3.12",
+  "Topic :: Software Development :: Quality Assurance",
 ]
-
 dependencies = [
   "toml>=0.10.2",
-  "pyfiglet"  
+  "pyfiglet>=0.8.0"  
```

Comment:
maybe it makes sense to have strict version dependencies or at least depends on the major version?